packages/@aws-cdk/aws-apigateway/lib/integrations/aws.ts

```diff
@@ -92,7 +92,8 @@ export class AwsIntegration extends Integration {
     });
   }
 
-  public bind(method: Method) {
+  public bind(method: Method): IntegrationConfig | undefined {
     this.scope = method;
+    return;
```

Comment:
Is this needed?

Reply:
Yes. The return type is no longer `void`.